Controller/DemoController.php

```diff
@@ -21,6 +21,130 @@
 class DemoController extends Controller
 {
     /**
+     * Displays the sub folder if it exists
+     *
+     * @param int $locationId of a folder
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function showSubFolderLinkAction( $locationId )
```

Comment:
```SubFolderLinkAction```? Link makes me thing in a link to a subfolder instead of o a list of subfolders...